### PR TITLE
AWS: Use persistent disk on master

### PR DESCRIPTION
**Pull request by justinsb**
@iterion Here is what I think we should do for the persistent disk on AWS.  Can you PTAL?

(I was going to submit a PR to your branch, but github won't let me for some reason)

If you are OK with it, not sure how exactly we would submit this (boring legal stuff).  I think you are still the author on the first commit, and you have signed the CLA, so I think clabot should be able to figure it out.

### CODE REVIEW

File: cluster/aws/util.sh [hunk 1]
```diff
@@ -798,8 +832,14 @@ function kube-up {
       KUBE_MASTER_IP=$(assign-elastic-ip $ip $master_id)
       echo -e " ${color_green}[master running @${KUBE_MASTER_IP}]${color_norm}"
 
-      # We are not able to add a route to the instance until that instance is in "running" state.
+      # We are not able to add a route or volume to the instance until that instance is in "running" state.
       wait-for-instance-running $master_id
+
+      # This is a race between instance start and volume attachment.  There appears to be no way to start an AWS instance with a volume attached.
+      # TODO: Wait for volume to be ready in setup-master-pd.sh
```
Comment (iterion) [inline cluster/aws/util.sh]: Is this still a TODO? It looks like we're actually waiting now.

**Comment by iterion** [comment]
LGTM minus my single comment.

Per @googlebot's request I can confirm I wrote the first commit and am OK with it being merged. :)

**Comment by justinsb** [comment]
Thanks @iterion.  Cleaned up the TODO (and rebased my commits to make it tidier)

**Comment by justinsb** [comment]
Rebased to try to get a shippable retest.

Anyone know if the removal of the cla:no tag is a manual process?

**Comment by thockin** [comment]
The CLA failure seems to be for @iterion - have you signed our CLA?

**Comment by iterion** [comment]
I have indeed, googlebot seems confused.
Other PR's of mine with `cla: yes`: https://github.com/GoogleCloudPlatform/kubernetes/pulls?q=is%3Apr+author%3Aiterion+is%3Aclosed

**Comment by thockin** [comment]
I can manually verify CLA and override bot.  Are you guys both LGTM on this?

**Comment by justinsb** [comment]
I am (though I wrote part of it): LGTM

**Comment by iterion** [comment]
LGTM too.

**Comment by a-robinson** [comment]
LGTM

**Comment by thockin** [comment]
LGTM OK to merge

**Comment by roberthbailey** [comment]
LGTM as well.

**Pull request merged**